Tests/Pcap++Test/Tests/FilterTests.cpp

```diff
@@ -814,4 +814,11 @@ PTF_TEST_CASE(TestPcapFilters_LinkLayer)
 	}
 	PTF_ASSERT_EQUAL(validCounter, 62);
 	rawPacketVec.clear();
+
+
+// pcpp::LINKTYPE_LINUX_SLL2 layer
+	pcpp::PcapFileReaderDevice fileReaderDev4(SLL2_PCAP_PATH);
+	PTF_ASSERT_TRUE(fileReaderDev4.open());
+	fileReaderDev4.getNextPackets(rawPacketVec);
+	fileReaderDev4.close();
```

Comment:
We should iterate over the packets in `rawPacketVec` and make sure all of them are of type SLL2 like we do in the other link types

Reply:
I ended up removing this test and adding another test instead to read and write pcap files with SLL2: https://github.com/seladb/PcapPlusPlus/pull/1138/commits/a90944e4bc8e60a00433c86536fc2465460c2408